homeassistant/components/lightwave/sensor.py

```diff
@@ -35,6 +35,11 @@ def __init__(self, name, lwlink, serial):
         self._lwlink = lwlink
         self._serial = serial
 
+    @property
+    def unique_id(self):
```

Comment:
Alternatively, we can set these features via class/instance attributes:
https://developers.home-assistant.io/docs/core/entity#entity-class-attributes

See the device class and unit of measurement features we've set above.

Reply:
Sure accepted proposed change